deploy/helm/charts/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v2
+name: zfs-localpv
+description: CSI Driver for dynamic provisioning of ZFS Persistent Local Volumes.
+version: 1.2.0
```

Comment:
1.2.1?

Reply:
version is helm chart version. The app version should match the underlying tag versions.